python/test/test_api.py

```diff
@@ -499,3 +645,54 @@ def test_parameters(self, server_options):
                 output_memory_type="cpu",
                 raise_on_error=True,
             )
+
+
+class TestMemoryLeak:
+    @staticmethod
+    @contextmanager
+    def _leak_detector(min_mem_inc, max_mem_inc):
+        gc.collect()
+        tracemalloc.start()
+        current, peak = tracemalloc.get_traced_memory()
+        yield
+        new_current, new_peak = tracemalloc.get_traced_memory()
+        mem_inc = new_current - current
+        assert mem_inc >= min_mem_inc
+        assert mem_inc < max_mem_inc
+        tracemalloc.stop()
+
+    # max_infer_repeat / min_infer_repeat >> max_mem_inc / min_mem_inc, so the memory
+    # increase is not proportional to the increase in inference requests.
+    @pytest.mark.parametrize("infer_repeat", [200, 500, 1000, 8000])
+    def test_inference_leak(self, server_options, infer_repeat):
+        min_mem_inc, max_mem_inc = 20000, 120000
```

Comment:
are these values in bytes? 20KB and 120KB?

Reply:
yes, those are in bytes, so 20KB and 120KB

Reply:
Can you add `_bytes` suffix to the var names?

Reply:
Sure, added the `_bytes` suffix. [Add _bytes suffix to mem size vars](https://github.com/triton-inference-server/core/pull/426/commits/536cf986c528229345406908f5b520b343061993)